tests/bluetooth/tester/src/l2cap.c

```diff
@@ -351,17 +351,21 @@ static void listen(uint8_t *data, uint16_t len)
 
 	server->accept = accept;
 	server->psm = cmd->psm;
-
-	if (server->psm == 0x00F4) {
-		/* TSPX_psm_encryption_key_size_required */
-		req_keysize = 16;
-	} else if (server->psm == 0x00F3) {
-		/* TSPX_psm_authentication_required */
-		authorize_flag = true;
-	} else if (server->psm == 0x00F2) {
+	LOG_DBG("sec lvl: %d, keysize: %d", cmd->security, cmd->key_size);
+	if (server->psm == 0x00F2 || cmd->security == 2) {
```

Comment:
I assume that the `server->psm` is some predefined value from a specification? 

Otherwise I suggest used named variables/#defines to denote the different PSMs. 

What does the value in `cmd->security` mean? Kinda looks like 2 mean authentication, and 3 means authorization, but it's hard to tell.

Reply:
agree, lets update bttester.h with proper defines named after btp.txt